notebooks/116-sparsity-optimization/116-sparsity-optimization.ipynb

```diff
@@ -8,7 +8,9 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # create a folder
</pre><p>Please use <code>cache_dir</code> parameter in the .from_pretrained() method to store the model in the specific location. This step can be removed at all. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/903/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
@AlexKoff88 I tried, it error-ed out. Also even it is successfully, the cache directory structure will look like git tree, we would need another few lines to obtain the openvino_model.xml and bin as well. imo, it is about the same as current version. The current version put all the related collaterals in one folder which is clearer. As far as the objective of the notebook is concerned, we already achieve it.

Reply:
I agree that the objective is achieved. I am just thinking that since the model is downloaded with `.from_pretrained()` method, there is no need to download it a second time. Moreover, the relative path in the `cache_dir` should be fixed in terms of structure. 
But if there are errors I am ok with this. Feel free to file an issue in Optimum.

Reply:
@AlexKoff88 I see where you are coming from now. `.from_pretrained()` will download a model to a cache folder regardless, `cache_dir` is for user customization but it doesn't work now. `hf_hub_download` doesn't download if a particular model has been downloaded (cached) before, it is also the direct/best way to return OpenVINO IRs, otherwise users would need to do some kind glob or folder walk etc. so, i think we are good now. )